website/docs/components/table/advanced-table/partials/guidelines/guidelines.md

```diff
@@ -167,6 +168,35 @@ Avoid using different density settings for parent and child rows.
 
 !!!
 
+### Expand/Collapse All Button
+The Expand/Collapse All button allows users to expand or collapse all rows, including nested rows. It provides quick access to more content but may impact readability when content is long or detailed.
+
+### Interactions
+
+#### Default state
+
+The Advanced Table supports any combination of expanded or collapsed rows on load. The button reflects the initial state.
+
+![Advanced Table with default height parent rows and short density nested rows.](/assets/components/table/advanced-table/expandable-rows-expand-all.png)
+<Doc::ImageCaption @text="Expand All if any rows are collapsed."/>
+
+![Advanced Table with default height parent rows and short density nested rows.](/assets/components/table/advanced-table/expandable-rows-collapse-all.png)
+<Doc::ImageCaption @text="Collapse All if all rows are expanded."/>
+
+### Collapsed state
+
+![Advanced Table with default height parent rows and short density nested rows.](/assets/components/table/advanced-table/expandable-rows-expand-state.png)
+<Doc::ImageCaption @text="Clicking Expand All expands all rows, including nested rows."/>
+
+### Expanded state
+
+![Advanced Table with default height parent rows and short density nested rows.](/assets/components/table/advanced-table/expandable-rows-collapse-state.png)
+<Doc::ImageCaption @text="Clicking Expand All expands all rows, including nested rows."/>
+
+### Mixed state 
+
+![Advanced Table with default height parent rows and short density nested rows.](/assets/components/table/advanced-table/expandable-rows-mixed-state.png)
+<Doc::ImageCaption @text="Clicking Expand All expands all rows, including nested rows."/>
```

Comment:
Suggestion: it seems a bit repetitive to have the same caption for all 3 images. Could they be removed if there is not anything else to add?